save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/demo/CpgView.kt

```diff
@@ -38,6 +39,15 @@ val cpgView: VFC = VFC {
         }
     }
     val (queryId, setQueryId) = useState("")
+    val (errorMessage, setErrorMessage) = useState("")
+    val errorWindowOpenness = useWindowOpenness()
+
+    displayInfoModal(
```

Comment:
Why do you use `displayInfoModal` for errors instead of infos?

Reply:
This is just error information. You need a window without additional actions and buttons. _displayInfoModal_ is great for this.

Reply:
Then it probably needs to be refactored into `displaySimpleModal`

Reply:
Ok, renamed